orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationError.java

```diff
@@ -18,6 +18,17 @@
 @Value
 @Beta
 public class OrchestrationError implements ClientError {
-  @Delegate(types = {ClientError.class})
   ErrorResponse originalResponse;
+
+  /**
+   * Gets the error message from the contained original response.
+   *
+   * @return the error message
+   */
+  @Nonnull
+  public String getMessage() {
+    return originalResponse.getCode() == 500
+        ? originalResponse.getLocation()
+        : originalResponse.getMessage();
```

Comment:
When the error code is 500, the `ClientResponseHandler` already adds `500 Internal Server Error` to the message, no need to repeat it.

Reply:
You shouldn't test bugs, btw here is the diff, nothing is obfuscated:
```diff
-Request failed with status 500 Internal Server Error and error message: 'Internal Server Error'
+Request failed with status 500 Internal Server Error and error message: 'Masking Module - Masking'
```

Reply:
Added test and message back